### DISCUSSION TITLE
The new Print icon in Duotone is awful when colored

### DISCUSSION THREAD

**Discussion by tinwe** [FortAwesome/Font-Awesome#19943] [General]
The new version of Print has a cut out in the middle of its pages, which only makes it confusing and misleading when colored. The previous version did not have this problem.

![Skärmbild 2023-10-12 080835](https://github.com/FortAwesome/Font-Awesome/assets/4280267/bac3b713-0426-40c6-8409-a432fd40264c)

**Comment by tagliala** [comment]
Thanks. Moved to an issue: #19944

If you have more details (version of FA without issues, first version with issues) please share there
